routes/api/v1/users.js

```diff
@@ -26,10 +26,10 @@ router.post("/", function(request, response, next){
     .then((validity) => {
       if (validity == false) {
         response.setHeader("Content-Type", "application/json");
-        response.status(500).send("This email is already in use!")
+        response.status(400).send("This email is already in use!")
       } else if (request.body.password != request.body.confirm_password) {
         response.setHeader("Content-Type", "application/json");
-        response.status(500).send("Passwords don't match")
+        response.status(400).send("Passwords don't match")
```

Comment:
This is a minor nitpick, but a `400` should generally be used if the request is the wrong format (like, if they sent a location instead of an email/password), and a `401` should be used for these sorts of errors where the request was structured correctly but not approved/authorized.

Reply:
Cool, will make the change. I wouldn't even call it a nitpick. The right way is the right way.